package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opsgenie-sdk",
-  "version": "0.4.5",
+  "version": "0.4.9",
   "lockfileVersion": 1,
```

Comment:
isn't path-parse a transitive dependency of jest? Its not advisable to update the package-lock.json directly.

Reply:
Actually, i have not updated the package-lock.json.

I followed the steps mentioned in veracode(VULN software). I ran npm shrinwrap which created new file shrinkwrap.json then updated the dependency. Removed the node modules and then ran npm install.

which updated the package-lock.json indirectly and i tried running all few test cases and they seem to be working fine